src/selection.coffee

```diff
@@ -174,6 +174,18 @@ class Selection extends Model
   intersectsScreenRow: (screenRow) ->
     @getScreenRange().intersectsRow(screenRow)
 
+  # Public: Identifies if this selection's rows intersects with another selection's rows.
```

Comment:
I'm not sure this belongs in the API. Mainly I'm hanging up on the `ByRow` part of the name feeling a little confusing. What if you just used an existing method in your editor code?

```coffee
selectionA.getScreenRange().intersectsRowRange(selectionB.getScreenRange().getRowRange()...)
```

It's a bit longer but it's nice that the API doesn't gain surface area.